CardMaker/CardMaker.csproj

```diff
@@ -537,12 +537,7 @@
   <PropertyGroup>
     <PreBuildEvent>
     </PreBuildEvent>
-    <PostBuildEvent>IF $(ConfigurationName) == Release (

```

Comment:
Can you add this back in? I should probably make it so that if 7z is missing it doesn't break. 🤔